assets/src/components/v2/elevator/closures.tsx

```diff
@@ -180,9 +199,11 @@ const sortStations = (
     }
   });
 
+type ClosuresStatus = "no_closures";
```

Comment:
Since this is only used in one place, I don't think we need a separate type definition for it; the prop could be defined as

```ts
stations_with_closures: StationWithClosures[] | "no_closures";
```

This way there is no cross-referencing needed to understand all possible values.

Reply:
I made this with the plan in mind of adding the "no relevant closures" case as another keyword (the other half of this ticket). That way the props def could remain `StationWithClosures[] | ClosuresStatus`. But I agree it isn't strictly necessary and maybe just adds boilerplate